README.md

```diff
@@ -147,7 +144,9 @@ region_kms_keys = {
 ```
 
 ```console
-AWS_PROFILE=cool-images-ec2amicreate-windows-commando-packer packer build --timestamp-ui -var-file release.pkrvars.hcl src/packer.pkr.hcl
+PKR_VAR_winrm_password="your-winrm-password" \
+AWS_PROFILE=cool-images-ec2amicreate-windows-server-packer \
+packer build --timestamp-ui -var winrm_password={password} \ -var-file release.pkrvars.hcl src/packer.pkr.hcl
```

Comment:
Similar change as above, except I removed the now-unnecessary `PKR_VAR_winrm_password` env variable, which I assume was an oversight in https://github.com/cisagov/windows-server-packer/pull/10/commits/46c6ab255bd389117c5663ab2b288b092cf80c4f:
```suggestion
AWS_PROFILE=cool-images-ec2amicreate-windows-server-packer \
packer build --timestamp-ui -var winrm_password="your-winrm-password" \
-var-file release.pkrvars.hcl src/packer.pkr.hcl
```

Reply:
@dav3r wasn't sure how I should represent an example password. I went with your suggestion. Resolved with [09eb2e6](https://github.com/cisagov/windows-server-packer/pull/10/commits/09eb2e6852084dffa68fddd605f151664e95cc86)

Reply:
@itsmostafa Please check again; https://github.com/cisagov/windows-server-packer/pull/10/commits/09eb2e6852084dffa68fddd605f151664e95cc86 does not remove this line (which it should):
```
PKR_VAR_winrm_password="your-winrm-password" \
```

Reply:
@dav3r resolved with [765a968](https://github.com/cisagov/windows-server-packer/pull/10/commits/765a968430a18b95b4fbd04acc92742dd01778b4)

Reply:
This looks good now, thanks!